test/unit/codegen/codegen_llvm_ir.cpp

```diff
@@ -1283,6 +1286,37 @@ SCENARIO("Vector library calls", "[visitor][llvm][vector_lib]") {
     }
 }
 
+//=============================================================================
+// Fast math flags
+//=============================================================================
+
+SCENARIO("Fast math flags", "[visitor][llvm]") {
+    GIVEN("A function to produce fma and specified math flags") {
+        std::string nmodl_text = R"(
+            FUNCTION foo(a, b, c) {
+                foo = (a * b) + c
+            }
+        )";
+
+        THEN("instructions are generated with the flags set") {
+            std::string module_string =
+                run_llvm_visitor(nmodl_text,
+                                 /*opt=*/true,
+                                 /*use_single_precision=*/false,
+                                 /*vector_width=*/1,
```

Comment:
Just out of curiosity - why you used serial kernel?

Reply:
We do not vectorize non-kernel functions, and here I was just testing that the flags are added (so purely semantic test)